services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/healthcheck/HealthCheckExtension.groovy

```diff
@@ -1,16 +1,18 @@
 package org.openkilda.functionaltests.extension.healthcheck
 
+import org.openkilda.functionaltests.extension.FeatureOrderExtension
```

Comment:
To be consistent with `perftests` it should be named `functests`. Or we can use `performancetests`, `functionaltests`.

Reply:
changed to performancetests